Makefile

```diff
@@ -364,7 +364,6 @@ $(POT): securedrop
 		--charset=utf-8 \
```

Comment:
Can the `VERSION=$(shell python -c "import securedrop.version; print(securedrop.version.__version__)")` from the top of the Makefile also be removed now?

Reply:
Good catch.  Removed in e3a534e9069906589a793179873328761254f493.